didx-xyz/yoma-mobile
# YOMA-525 - feat(UserQualifications): add credentials response to state

## Scope [YOMA-525](https://yomaworld.atlassian.net/browse/YOMA-525)

Get UserQualification data into state

![](https://media.giphy.com/media/26FPDJ6ZRqCHf3F1C/giphy.gif)
## Work Done
adding reducers and middleware to setup a UserQualifications state property and get the required data from the UserCredentialsSuccess payload.

## Steps to Test
run tests
run app and see that userQualifications exists (And is populated if you have userQuals)
